These headers are no longer provided by make headers_install.
--- sysklogd-1.4.1/module.h.orig	2006-09-02 17:15:40.000000000 +0200
+++ sysklogd-1.4.1/module.h	2006-09-02 17:15:44.000000000 +0200
@@ -28,8 +28,6 @@
  *	recent content from Linux 2.4/2.6.
  */
 
-#include <asm/atomic.h>
-
 #define MODULE_NAME_LEN 60
 
 struct kernel_sym
--- sysklogd-1.4.1/ksym_mod.c.orig	2006-09-02 17:16:05.000000000 +0200
+++ sysklogd-1.4.1/ksym_mod.c	2006-09-02 17:16:16.000000000 +0200
@@ -99,7 +99,6 @@
 #include <linux/time.h>
 #include <linux/linkage.h>
 #else /* __GLIBC__ */
-#include <linux/linkage.h>
 extern __off64_t lseek64 __P ((int __fd, __off64_t __offset, int __whence));
 extern int get_kernel_syms __P ((struct kernel_sym *__table));
 #endif /* __GLIBC__ */
